controllers/bentodeployment_controller.go

```diff
@@ -1702,6 +1710,33 @@ func (r *BentoDeploymentReconciler) generatePodTemplateSpec(ctx context.Context,
 		}
 	}
 
+	if need_monitor_container {
+		monitoring_config_template := `
+monitoring.enabled=true
+monitoring.type=otlp
+monitoring.options.endpoint=http://127.0.0.1:{%d}
+monitoring.options.insecure=true
+`
+		var bentoml_options string
+		var index int
+		for i, env := range envs {
+			if env.Name == "BENTOML_CONFIG_OPTIONS" {
+				bentoml_options = env.Value
+				index = i
+				break
+			}
+		}
+		if !strings.Contains(bentoml_options, "monitoring") {
+			bentoml_options = strings.TrimSpace(bentoml_options)
+			bentoml_options += fmt.Sprintf(monitoring_config_template, monitorExporterPort)
+		}
+		envs = append(envs[:index], envs[index+1:]...)
```

Comment:
If BENTOML_CONFIG_OPTIONS does not exist in the environment variables, is the first environment variable lost?

Reply:
Just pushed a new commit to fix it